gef-ui/templates/partials/header-component.njk

```diff
@@ -0,0 +1,51 @@
+            </svg>
+            <!--<![endif]-->
+            <!--[if IE 8]>
+            <img src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image" alt="GOVUK logo" width="36" height="32">
+            <![endif]-->
+            <span class="govuk-header__logotype-text">
+              GOV.UK
+            </span>
+          </span>
+        </a>
+      </div>
+      <div class="govuk-header__content">
+        <a href="/dashboard" class="govuk-header__link govuk-header__link--service-name" data-cy="serviceName-header">
+          Get a guarantee for export finance
+        </a>
+        <nav aria-label="Menu" class="govuk-header__navigation" data-cy="navigation-header">
+          <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide menu">Menu</button>
+          <ul id="navigation" class="govuk-header__navigation-list">
+            <li class="govuk-header__navigation-item govuk-header__navigation-item--active">
+              <a class="govuk-header__link" href="/user/{{user._id}}" data-cy="username-header">
+                {{user.username}}
+              </a>
+            </li>
+            <li class="govuk-header__navigation-item">
+              <a class="govuk-header__link" href="/user/{{user._id}}" data-cy="profile-header">
+                Profile
+              </a>
+            </li>
+            <li class="govuk-header__navigation-item">
+              <a class="govuk-header__link" href="/logout" data-cy="logout-header">
+                Sign out
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </header>
+
+{% endmacro %}
```

Comment:
What's the difference between this component and the no-user version, can they be combined?

Reply:
Nope, because can't have html if statements, no user (logged out) has no navigational component with the name profile and log out etc so need 2 different files

Reply:
in a partial we should be able to add an if/else statement. it's a .njk file after all. in that case, the 2 files could be combined into 1, with an if/else statement to check whether the user is logged in or not

Reply:
done